src/loaders/api/apiAsync.js

```diff
@@ -0,0 +1,96 @@
+import { FEATURE_NAMES } from "../features/features"
```

Comment:
this is the portion of the API that can be loaded async, as is what happens in current agent